nucliadb/nucliadb/ingest/orm/knowledgebox.py

```diff
@@ -119,7 +119,6 @@ async def exist_kb(cls, txn: Transaction, uuid: str) -> bool:
     async def delete_kb(cls, txn: Transaction, slug: str = "", kbid: str = ""):
         # Mark storage to be deleted
         # Mark keys to be deleted
-        logger.info(f"Deleting KB kbid={kbid} slug={slug}")
```

Comment:
I wonder if we should return an error in the event of the passed but not not found?
For if this happens again in the future

Reply:
I think we should rethink the entire KB slugs thing